wzdx/monitoring/metrics.py

```diff
@@ -0,0 +1,22 @@
+from google.api import label_pb2 as ga_label
+from google.api import metric_pb2 as ga_metric
+from google.cloud import monitoring_v3
+
+client = monitoring_v3.MetricServiceClient()
+project_name = f"projects/cdot-rtdh-dev"
```

Comment:
If we plan to allow this to run in test/prod we'll want to pull this from an environment variable rather than hard-coded.

Reply:
this script is only run once, to create the metrics. I just ran it locally